renku/cli/_providers/zenodo.py

```diff
@@ -214,9 +214,11 @@ def version(self):
     def is_last_version(self, uri):
         """Check if record is at last possible version."""
         if is_doi(uri):
+            breakpoint()
```

Comment:
😎

Reply:
Thank you for reviewing WIP. I push the last nite to save my work, was too tired to clean up properly. :clinking_glasses: